pkg/api/redistribution.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/tracing"
+	"net/http"
+)
+
+type NodeStatusResponse struct {
+	State  string `json:"state"`
+	Round  uint64 `json:"round"`
+	Block  uint64 `json:"block"`
+	Reward string `json:"reward"`
+	Fees   string `json:"fees"`
+}
+
+func (s *Service) redistributionStatusHandler(w http.ResponseWriter, r *http.Request) {
+	logger := tracing.NewLoggerWithTraceID(r.Context(), s.logger.WithName("redistribution_status").Build())
```

Comment:
The pattern for naming logger handlers is **method_endpointname** so in this case it will be `get_redistributionstate`.